account_invoice_merge/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,8 @@
+This module adds an action in the invoices list to merge invoices. Here are
+the conditions to allow merging:
+
+* Type should be the same (Customer Invoice, Supplier Invoice, Customer Refund or
+  Supplier Refund)
+* Partner should be the same
+* Currency should be the same
+* Account receivable account should be the same
```

Comment:
reading the code there are more constraints to be added (company, currency, journal)